contracts/bridge/x/v4/Core.sol

```diff
@@ -0,0 +1,269 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
```

Comment:
I assume this contract is copied from somewhere. Please put a comment at the top to explain what this contract is and provide a link to where it has been copied from.

Reply:
👍 This was generated from the ABI json that Starkware provided us.